package.json

```diff
@@ -1,42 +1,54 @@
 {
-  "name": "ovh-angular-responsive-page-switcher",
+  "name": "@ovh-ux/ng-ovh-responsive-page-switcher",
   "version": "1.1.1",
   "description": "It detects available width for displaying page in 2 distinct modes",
-  "main": "./dist/ovh-angular-responsive-page-switcher.min.js",
-  "repository": "ovh-ux/ovh-angular-responsive-page-switcher",
-  "author": "OVH SAS",
+  "keywords": [
+    "angular",
```

Comment:
keywords aren't related to this package.

We can use the following list:
- page
- responsive
- switcher
- angularjs
- ovhcloud